cmd/kubent/main.go

```diff
@@ -70,6 +75,7 @@ func initCollectors(config *config.Config) []collector.Collector {
 }
 
 func main() {
+	exitCode := EXIT_CODE_SUCCESS
```

Comment:
I think it should be 200, by default incase of silient errors :)

Reply:
So we set exit code to be 1 (generic error) by default & only change it to 0 or 200 based on the flag if we successfully get to the end.